### PR TITLE
Update Membership Page with Platinum Model

### PR DESCRIPTION
**Pull request by ErinBecker**
DO NOT MERGE until approved by @kmomar. @kmomar - please review carefully, as there are a lot of changes here. Some are cosmetic, but the main update is to communicate clearly about the new Platinum membership model that @kariljordan and I have developed. This should be released immediately before the corresponding blog post, so that this link is live before the blog post is visible.

### CODE REVIEW

**Comment by elletjies** [comment]
@ErinBecker I am using this document https://docs.google.com/spreadsheets/d/1dbAnIRxjoArVqpKuwmli2glYzcz_LTw3w0WdpxiUPtY/edit#gid=1045282279 to review. 

online with one in-house Instructor Trainer, coordinated by The Carpentries = $750 each ** should this not be $500
online or in-person with two in-house Instructor Trainers, coordinated by member organisation = $500 each **should this not be $750? Where do I find the other information?

File: pages/membership.md [hunk 1]
```diff
@@ -5,36 +5,77 @@ title: Become a Member Organisation
...
-|Discount for additional <br>coordinated workshops|20%|33%|50%|negotiable|
-|Self-organized workshops<br> at member organisation **|no-charge|no-charge|no-charge|no-charge|
-|Number of instructors <br>trained ***|0|6 online|15 with possibility <br>for in-person^<br>training event|negotiable|
-|Seat on the <br>Carpentries Member Council|Yes|Yes|Yes|Yes|
-|Train an in-house instructor <br>trainer at member org|No|No|No|Available|
-|Lesson development <br>services|No|No|No|Available|
-|Membership Dues (annual)|[$5,000](mailto:memberships@carpentries.org?subject=Interested in Bronze Membership)|[$7,500](mailto:memberships@carpentries.org?subject=Interested in Silver Membership)|[$15,000](mailto:memberships@carpentries.org?subject=Interested in Gold Membership)|[Contact us](mailto:memberships@carpentries.org?subject=Interested in Platinum Membership)|
+|Number of coordinated workshops|2|4|6|6|
+|Discount for additional <br>coordinated workshops|20%|33%|50%|50%|
+|Self-organized workshops**|no-charge|no-charge|no-charge|no-charge|
```
Comment (kmomar) [inline pages/membership.md]: Change to self-organised

File: pages/membership.md [hunk 2]
```diff
@@ -5,36 +5,77 @@ title: Become a Member Organisation
...
 <a href="https://carpentries.typeform.com/to/Hmfe6L">
   <button class="btn">
     Join Now
   </button>
 </a>
 
-#### Frequently Asked Questions
+### Platinum Memberships
+
+The Platinum tier is our most customizable membership. The price of a Platinum membership
```
Comment (kmomar) [inline pages/membership.md]: customisable

**Comment by ErinBecker** [comment]
> @ErinBecker I am using this document https://docs.google.com/spreadsheets/d/1dbAnIRxjoArVqpKuwmli2glYzcz_LTw3w0WdpxiUPtY/edit#gid=1045282279 to review.
> 
> online with one in-house Instructor Trainer, coordinated by The Carpentries = $750 each ** should this not be $500
> online or in-person with two in-house Instructor Trainers, coordinated by member organisation = $500 each **should this not be $750? Where do I find the other information?

If they have two in-house Trainers, they are able to fully cover the teaching of the event, therefore they use fewer Carpentries resources and don't cost us as much money. So the $500 price for when they have two in-house trainers is correct.

**Comment by AngeliqueV** [comment]
> > @ErinBecker I am using this document https://docs.google.com/spreadsheets/d/1dbAnIRxjoArVqpKuwmli2glYzcz_LTw3w0WdpxiUPtY/edit#gid=1045282279 to review.
> > online with one in-house Instructor Trainer, coordinated by The Carpentries = $750 each ** should this not be $500
> > online or in-person with two in-house Instructor Trainers, coordinated by member organisation = $500 each **should this not be $750? Where do I find the other information?
> 
> If they have two in-house Trainers, they are able to fully cover the teaching of the event, therefore they use fewer Carpentries resources and don't cost us as much money. So the $500 price for when they have two in-house trainers is correct.

Then I am all good!

**Comment by maneesha** [comment]
@kmomar @ErinBecker  is this ready to merge?

**Comment by ErinBecker** [comment]
@maneesha - Please don't merge! We're going to be putting this out in conjunction with a blog post and targeted emails. @kmomar will do the merge on this one at the same time as the blog post.

**Comment by ErinBecker** [comment]
@kmomar - I've incorporated all of your great suggestions from our conversation this morning. Please take a look at your email with an additional question about clarifying an aspect of this model raised by @Talishask.

**Comment by kariljordan** [comment]
This page is wonderful, and @kmomar's recommendations made the information even more clear. 

@ErinBecker Is there any place we should/could add a disclaimer that makes it clear that we reserve the right to deny a platinum membership request? For example, if we already have enough Trainers in a particular region and are not in a position to provide a discount for another in-house trainer, we should be able to communicate that. I'd like to set that expectation up front if at all possible.

**Comment by ErinBecker** [comment]
@kariljordan  I added “Discounts listed above for are limited and available only in cases where a potential member organisation’s ability to offer support to Instructor Trainers, Regional Coordinators, and/or Community Leaders is in line with The Carpentries needs for support in a particular geographic region. We reserve the right to award these discounts as appropriate for our organisational growth and sustainability.” Please feel free to adjust as you see fit!

**Comment by kariljordan** [comment]
@kmomar I have reviewed this PR and it is ready to go.

**Pull request merged**